projects/ADuCM3029_demo_cn0326/src/AD7793.c

```diff
@@ -155,10 +155,7 @@ uint32_t AD7793_Scan(enMode mode,  uint8_t ui8channel)
 {
 	static  uint32_t ui32result, ui32reg_value;
 
-	/* Set value (read command + DATA register address) to write in COMM
-	 * register */
-	uint8_t ui8reg_adrr = (AD7793_COMM_READ
-	                       | AD7793_COMM_ADR(AD7793_REG_DATA));
```

Comment:
this looks incorrect; you are removing the address value here; this will likely not compile;
it would be a good idea to re-order this commit after commit 6210d47cdd335d0458fed05efaadbf3ba1ec3ff5 ( cn0326: replace spi read function with wrapper function  )
but for now, i guess we could leave it like this